### DISCUSSION TITLE
What is the latest version of sonic worlds next that works in godot 3?

### DISCUSSION THREAD

**Discussion by hardcaz** [Techokami/SonicWorldsNext#43] [Q&A] [answered]
Asking because my computer cant run godot 4

**Comment by DimensionWarped** [comment]
master is currently on Godot 3. Not sure when it'll get moved to a legacy branch and the Godot 4 branch will take its place, but for now just use master.

**Comment by Techokami** [comment, marked as answer]
Godot 3 support is available from the "Sonic-Worlds-Next-Godot-3" branch, which is considered legacy and will not be getting much in the way of updates.

**Comment by Techokami** [comment]
Also, just as a question, what exactly is preventing your computer from using Godot 4? I'm not aware of any minimum system requirement changes...?

**Reply by hardcaz** [reply]
My pc can only handle up to opengl 3.1 (on windows atleast) (blame me only having a intel cpu)

**Reply by Techokami** [reply]
Only integrated graphics? Owch O_o my condolences

**Reply by hardcaz** [reply]
Yeah (wouldnt be a problem if godot 4 had es2 support)

**Reply by Techokami** [reply]
Hey, we might have a way to get Godot 4 working on your older machine? There is a Compatability mode that uses the same base requirements as Godot 3. The latest commit switches the renderer to it; can you give it a shot on your machine? We'd love to make this thing as compatible as possible.

**Reply by hardcaz** [reply]
Is the compatibility mode the opengl 3.3 mode?

If so then no it wouldnt work

**Reply by Renhoex** [reply]
according to the blog post

https://godotengine.org/article/rendering-priorities-4-1/

"Godot 4.0 shipped with a low-end focused renderer which targets OpenGL 3.3 / OpenGL ES 3.0 / WebGL 2 devices. The GL Compatibility renderer (as we call it) is written with low-end and mobile devices in mind and should run much more efficiently on those devices than the GLES3 renderer in Godot 3.x."

I don't know if your machine supports OpenGL ES 3.0 but from the sounds of it it doesn't look like it'd work.

**Reply by hardcaz** [reply]
opengl ES 3.0 doesnt work due to it being based on opengl 3.3

Atleast on windows

**Reply by hardcaz** [reply]
ANGLE works but the text is too big (unsure why)
